docs/concepts.md

```diff
@@ -25,11 +25,19 @@ on the standard Config class if nothing is found in the database.
 ## User Providers
 
 You can use your own models to handle user persistence. Shield calls this the "User Provider" class. A default model
-is provided for you at `CodeIgniter\Shield\Models\UserModel`. You can change this in the `Config\Auth::$userProvider` setting.
+is provided for you at `CodeIgniter\Shield\Models\UserModel`. You can change this in the `Config\Auth->userProvider` setting.
```

Comment:
I feel `Config\Auth::$userProvider` is better than `Config\Auth->userProvider`.

Reply:
Wait. I thought I changed this already during the conflict resolution?

Reply:
Fixed.